src/charm.py

```diff
@@ -107,13 +107,7 @@ def __init__(self, *args):
             ],
         )
         self._grafana_dashboard_provider = GrafanaDashboardProvider(charm=self)
-        self._log_proxy = LogProxyConsumer(
-            charm=self,
-            relation_name="logging",
-            log_files=[self._log_path],
-            container_name=self._container_name,
-            enable_syslog=False,
-        )
+        self._log_forwarding = LogForwarder(self, relation_name="logging")
```

Comment:
Does this mean we need to `assumes juju >= 3.4`?

Reply:
Done!